### ISSUE TITLE
Too long movie titles appear to break the UI

### ISSUE THREAD

**Issue by HavermansStef** [jellyfin/jellyfin-web#1418]
**Describe the bug**
I've come to notice way too long movies titles break the UI.

**To Reproduce**
1. Add a movie with a long title such as Jim & Andy: The Great Beyond- Featuring a Very Special, Contractually Obligated Mention of Tony Clifton
2. Navigate to the movie's item details page.

**Expected behavior**
There is an example picture of the UI when the title is shorter. I would suggest putting the overflow below the title or having dynamic sizing of the div that holds the title.
It appears the item also holds the text-overflow:ellipsis, which doesn't appear to be working.

**Screenshots**
I've uploaded a picture of a movie where the title is short enough and one where the title is too long.
https://imgur.com/gallery/IV0hy0d

**System (please complete the following information):**
 - OS: [Docker in Synology]
 - Browser: [Firefox, Chrome]
 - Jellyfin Version: [e.g. 10.5.5]

**Predicted Solution**
It can be fixed by giving detailPagePrimaryContainer a fixed height, such as 100px.
https://imgur.com/gallery/kynmqP2

I can make a PR for this, given it's approval by the devs. It would be the first time I help in this project, so if there is something else, like adding myself to the contributors, etc that needs to be done first, feel free to let me know.

**Comment by HavermansStef** [comment]
Duplicate of https://github.com/jellyfin/jellyfin-web/issues/1385

**Comment by Influence365** [comment]
Hopefully https://github.com/jellyfin/jellyfin-web/pull/1206 will fix this. Only issue with setting the container as a fixed height is that it makes the main detail buttons (play/trailer/more details ect. ) inaccessible.

Also, It looks like overflow: hidden; needs to be added for text-overflow:ellipsis to work
